### ISSUE TITLE
different animation effect Issue on multiple devices

### ISSUE THREAD

**Issue by kimyounghoons** [DanielMartinus/Konfetti#314]
When testing on multiple devices, there is a difference between 120 Frame device and 60 Frame device animation. How should I solve it?

**Comment by DanielMartinus** [comment]
There's probably an issue where the deltatime is not correctly applied to one of parameters. I should dive into this and see what's causing this.

**Comment by omargda** [comment]
I also observed differences in animation between a Samsung device and Google Pixel device. Samsung shoots down (as desired), but on a Pixel it shoots to the right

**Comment by DanielMartinus** [comment]
@omargda What configuration did you use? Can you share some more details about this specific case

**Comment by kit0kat** [comment]
Hi, I'm facing the same issue.

I'm using this configuration:

```
   val leftParty = Party(
        spread = 45,
        angle = 315,
        position = Position.Relative(0.0, 0.35),
        emitter = Emitter(duration = 3, TimeUnit.SECONDS).perSecond(30)
    )
    val rightParty = Party(
        spread = 45,
        angle = 225,
        position = Position.Relative(1.0, 0.35),
        emitter = Emitter(duration = 3, TimeUnit.SECONDS).perSecond(30)
    )
// ...
KonfettiView(
  modifier = Modifier.fillMaxSize(),
  parties = listOf(leftParty, rightParty),
)
```
It looks like this:

![image](https://github.com/DanielMartinus/Konfetti/assets/57002245/a97d17b2-39bf-4f43-bec9-27b4d3320676)

60 Hz is fine but 120 Hz seems broken. It looks like the confetti cannon only has half power at 120 Hz haha.

**Comment by DanielMartinus** [comment]
Thanks for the example! I'll make the proper adjustments. Right now the refresh rate seems to be locked at 60fps in the code. I'll make the proper adjustments and update the frame rate dynamically.

**Comment by DanielMartinus** [comment]
Opened a branch with a fix here: https://github.com/DanielMartinus/Konfetti/issues/314

@kit0kat would you be able to check-out the repo and run the xml or compose example on your device with 120Hz?

**Comment by kit0kat** [comment]
@DanielMartinus 
I checked out your branch and tried both XML and Compose examples.
It now works perfectly at 120 Hz 👍You are amazing.
When will the update be available?

**Comment by DanielMartinus** [comment]
@kit0kat I've just released 2.0.4 that includes the fix

**Comment by kit0kat** [comment]
Confirmed, fixed in production.
Thank you!!
